mcp/include/star_camera_receive.h

```diff
@@ -0,0 +1,17 @@
+/* command_list.h: BLAST command specification file definitions
```

Comment:
Assuming this is a new file and not a relocated one, header legalese is not accurate (filename, date)